libasn1compiler/asn1c_misc.c

```diff
@@ -61,8 +61,13 @@ asn1c_make_identifier(enum ami_flags_e flags, asn1p_expr_t *expr, ...) {
 		/*
 		 * Estimate the necessary storage size
 		 */
-		if(expr->Identifier == NULL)
-			return "Member";
+		if(expr->Identifier == NULL) {
+			if(storage_size < 10) {
```

Comment:
10 is not sufficient to store `MemberNNNNNNNNNNN\0`. I think you mean 20 here (which is what you used below).

Reply:
Thank you for reviewing !

It was almost 2:00 am at my place so I don't know what I was thinking then. 😢